src/components/TeachingBubble/TeachingBubble.scss

```diff
@@ -30,6 +30,7 @@
   }
   .ms-TeachingBubble-header--small & {
     @include ms-font-m;
+    @include margin-right(10px);
```

Comment:
I *think* this might still make sense, however, but i'm not sure.

Can you revert the button change, and then let me know if you want this still? Or close the PR.